src/pages/home/report/ReportActionCompose.js

```diff
@@ -1112,7 +1154,10 @@ class ReportActionCompose extends React.Component {
                                                     <PopoverMenu
                                                         animationInTiming={CONST.ANIMATION_IN_TIMING}
                                                         isVisible={this.state.isMenuVisible}
-                                                        onClose={() => this.setMenuVisibility(false)}
+                                                        onClose={() => {
+                                                            this.setMenuVisibility(false);
+                                                            this.focus(true);
```

Comment:
We should keep original focus state before opening popover.
We should not change current focus behavior. Please compare this branch build with main or staging.
Only the difference here should be to fix keyboard flash. Other behaviors should keep the same on all platforms.

https://github.com/Expensify/App/assets/108292595/ef1ab7d2-7ae8-4ee7-9b04-e749b98ee2bc

Reply:
@trjExpensify can you please confirm this is the new expected behavior we landed?
I see that composer is always focused upon modal close no matter keyboard was shown or not before opening modal

Reply:
>Yeah, we should be consistent and give focus back to the composer input here when the action menu closes, allowing the software keyboard to be shown at the same time.

To clarify what I was talking about here in the linked comment referenced is the scenario in which we were addressing in the issue.

> 1. The keyboard is open because you focused the composer
> 2. You tap + > add attachment 
> 3. Tap off the action menu
> 4. The action menu closes and the keyboard reopens

I agree that should be what we're shooting for here.

Reply:
Correct!